### PR TITLE
[DAT-16600] Implement method to send ping signal to DB server for connect command

### PR DESCRIPTION
**Pull request by vitaliimak**

### CODE REVIEW

**Review by suryaaki2 — APPROVED** [review]

File: src/main/java/liquibase/ext/mongodb/database/MongoLiquibaseDatabase.java
Comment (suryaaki2) [inline src/main/java/liquibase/ext/mongodb/database/MongoLiquibaseDatabase.java:128]: This should work for AWS DocumentDB as well?

**Review by suryaaki2 — CHANGES_REQUESTED** [review]

File: src/main/java/liquibase/ext/mongodb/database/MongoLiquibaseDatabase.java
Comment (vitaliimak) [reply src/main/java/liquibase/ext/mongodb/database/MongoLiquibaseDatabase.java:128]: Yes, it should work. At least, it works for Localstack DynamoDB.

**Review by filipelautert — APPROVED** [review]

**Comment by vitaliimak** [comment]
The liquibase PR is merged, but this repository uses stable liquibase as a dependency, so we should wait for the new liquibase release before merging this PR.
https://github.com/liquibase/liquibase-mongodb/actions/runs/8599495602/job/23562516426?pr=517#step:10:1714

**Review by KushnirykOleh — APPROVED** [review]

**Review by suryaaki2 — APPROVED** [review]

**Pull request merged**
